### PR TITLE
[Issues] templates/*: improve semantic html and add missing heading headings f…

### PR DESCRIPTION
**Pull request by phillimorland**
…ixes #1870 also fixes some a11y for the new tiles

### CODE REVIEW

**Comment by phillimorland** [comment]
@fuzzylogic2000 there's some weird stuff going on with the abbreviated times but when looking at it I didn't understand the different indexes when referencing the same time?! maybe we can look at it together or you can add a comment explanation?

File: apps/projects/templates/a4_candy_projects/includes/project_timeline-carousel.html [hunk 1]
```diff
@@ -14,11 +14,11 @@
                         <div class="timeline-carousel__label {% if forloop.counter0 == initial_slide %} initial {% endif %}">
                             <span id="timeline_entry_{{ forloop.counter }}">
                                 {% if  date.type == 'module' %}
-                                    <span class="timeline-carousel__title">{{ date.title }}</span><br>
-                                    {% trans 'from' %} {{ date.date | date }}
+                                    <h2 class="timeline-carousel__title">{{ date.title }}</h2>
+                                    <date>{% trans 'from' %} {{ date.date | date }}</date>
```
Comment (fuzzylogic2000) [inline apps/projects/templates/a4_candy_projects/includes/project_timeline-carousel.html:18]: Not new, but seeing this: we should probably use blocktranslate here.

File: apps/projects/templates/a4_candy_projects/project_detail.html [hunk 2]
```diff
@@ -161,6 +161,8 @@ <h3 class="u-top-divider">{% trans 'Contact for questions' %}</h3>
                 <div class="timeline-carousel__wrapper">
                 {% if project.display_timeline %}
                     {% include "a4_candy_projects/includes/project_timeline-carousel.html" with dates=project.participation_dates project=project %}
+                {% else %}
+                    <h2 class="visually-hidden">{% translate 'Online articipation' %}</h2>
```
Comment (fuzzylogic2000) [inline apps/projects/templates/a4_candy_projects/project_detail.html:165]: There is a p missing here 'Online participation'

**Review by fuzzylogic2000 — CHANGES_REQUESTED** [review]
Looks good! Only one typo that needs to be fixed first.
I will add an explanation for the ebbreviated stuff and the blocktranslate in an extra branch.

**Review by fuzzylogic2000 — APPROVED** [review]
Noice!

**Pull request merged**